### ISSUE TITLE
playback loop enhancement suggestion

### ISSUE THREAD

**Issue by roian** [kristinbranson/APT#327]
I would prefer if the playback loop ended at the frame i'm having trouble with, so I could "follow" the difficult body part in when it is occluded.  Would it be possible to have a toggle (could be under navigation preferences) that would change where the playback loop starts/ends?  e.g. "start with current frame", "end with current frame", "centered on current frame" (this last is what it does now).

**Comment by arobie** [comment]
That’s a good idea

**Comment by allenleetc** [comment]
Yes this should be easy, if it causes significant pain please bump

**Comment by roian** [comment]
probably not a priority now, i just wanted to write it down whilst i was thinking on it...
